docs/before_you_begin.rst

```diff
@@ -1,39 +1,45 @@
-Foundation <https://securedrop.org/help>`__ for installation assistance.
-
-Before you begin, you will need to assemble all the :doc:`hardware <hardware>`
-that you are going to use.
-
-To better assist in the installation process, you can organize your work using the
-`SecureDrop Installation Worksheet <https://docs.google.com/a/freedom.press/document/d/18RMAzhx1XCgpmw366I8tItBXQTzkFy_i_D0c605DTS8/edit?usp=sharing>`__.
-It is **critical** you destroy this worksheet when your installation is complete
-and all of your passphrases have been safely stored in a password manager.
-
-When running commands or editing configuration files that include
-filenames, version numbers, usernames, and hostnames or IP addresses,
-make sure you use the appropriate values for your instance.
-
-Once you're familiar with SecureDrop, you've made your plan, your
-organization is ready to follow-through and you have the required
-hardware assembled before you, you're ready to begin.
+Before you get started, read the :doc:`Overview <overview>` and familiarize
+yourself with the :doc:`terminology<terminology>` and the :doc:`passphrases
+<passphrases>` involved in SecureDrop's operations. You may wish to leave these
+documents open in other tabs for reference as you work.
+
+SecureDrop is a technical tool. It is designed to protect journalists and
+sources, but no tool can guarantee safety. This guide will instruct you in
+installing and configuring SecureDrop, but it does not explain how to use it
+safely and effectively. Put another way: at the end of this guide, you will have
+built a car; you will not know how to drive. The :ref:`Deployment Guide
+<deployment>` contains best practices for working with SecureDrop. Make sure to
+read it after completing the installation.
+
+Installing SecureDrop is an extended manual process which requires extensive
+preparation, equipment, and organizational support. You will likely need to set
+aside a day to complete the install process. Installing SecureDrop requires an
+admin with at least basic familiarity with Linux, the GNU core utilities, and
+Bash shell. If you are not proficient in these areas, it is strongly recommended
+that you contact the `Freedom of the Press Foundation
+<https://securedrop.org/help>`__ for installation assistance.
+
+Before you begin, you will need to assemble the required :doc:`hardware <hardware>`.
+
+To assist in the installation process, we offer a `SecureDrop Installation
+Worksheet`_.  It is **critical** that you destroy this worksheet when your
+installation is complete and all of your passphrases have been safely stored in
+a password manager.
+
+When running commands or editing configuration files that include filenames,
+version numbers, usernames, hostnames, or IP addresses, make sure to use the
+appropriate values for your instance.
+
+Once you're familiar with SecureDrop, have made your plan, ensured your
+organization is ready to follow through, and assembled the necessary hardware,
```

Comment:
Nit: we have **required hardware** above, is there a reason to change to **necessary** here?